src/components/structures/RoomSearchView.tsx

```diff
@@ -159,7 +159,7 @@ export const RoomSearchView = forwardRef<ScrollPanel, Props>(
         }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
         // show searching spinner
-        if (results?.count === undefined) {
+        if (results === null) {
```

Comment:
this looks sensible.

But: how does it differ from the spinner shown when `inProgress` is true?

(Maybe we can rip out the inProgress stuff?)

Reply:
inProgress is used for any loading including pagination, this is for the initial load. Not sure why its not just the same spinner but would rather not make aesthetic changes here.